packages/tinacms/src/toolkit/fields/plugins/blocks-field-plugin/index.tsx

```diff
@@ -197,7 +197,7 @@ const BlockListItem = ({
   return (
     <Draggable
       key={index}
-      type={field.name}
+      // type={field.name}
```

Comment:
**question**: Why is this commented-out code retained?

Reply:
Removed